eox_core/cms/templates/management.html

```diff
@@ -1,23 +1,29 @@
 <%inherit file="base.html" />
 <%namespace name='static' file='static_content.html'/>
 <%!
-  from openedx.core.djangolib.js_utils import dump_js_escaped_json
+  from openedx.core.djangolib.js_utils import dump_js_escaped_json, js_escaped_string
 %>
 <%block name="title">${"Course management"}</%block>
 <%block name="header_extras">
 
 </%block>
 
 <%block name="content">
-    <link type='text/css' rel='stylesheet' href='${STATIC_URL}css/vendor/bootstrap.min.css'>
-    <div class="container" style="font-size: 2em;">
+    <link type='text/css' rel='stylesheet' href='${STATIC_URL}bundle/css/course-management.bundle.css'>
```

Comment:
normally on the Open edX platform the css static files are loaded using:
`${static.url('xxx/yyy/zzz/file.min.css')}`

Why are we not using that standard here?